crates/ruff/src/rules/flake8_todo/rules.rs

```diff
@@ -0,0 +1,170 @@
+use std::iter::Peekable;
+
+use once_cell::sync::Lazy;
+
+use regex::{CaptureMatches, Regex};
+use ruff_diagnostics::{Diagnostic, Violation};
+use ruff_macros::{derive_message_formats, violation};
+use ruff_python_ast::types::Range;
+use rustpython_parser::ast::Location;
+use rustpython_parser::lexer::LexResult;
+use rustpython_parser::Tok;
+
+#[violation]
+pub struct InvalidTODOTag {
+    pub tag: String,
+}
+
+// TODO - autofix this to just insert TODO instead of the tag?
```

Comment:
Do we actually want to do this? If not, I'll remove this comment when I implement T006.